tests/unittests/DriverTests.cpp

```diff
@@ -3,6 +3,7 @@
 
 #include "Test.h"
 #include <fmt/core.h>
+#include <iostream>
```

Comment:
Why was this extra include added?

Reply:
Ups... Added a `cout` to get the output of a tests and didn't remember to remove it. Fixed :)